modyn/config/schema/pipeline.py

```diff
@@ -101,8 +101,13 @@ class PresamplingConfig(BaseModel):
 class DownsamplingConfig(BaseModel):
     """Config for the downsampling strategy of SelectionStrategy."""
 
-    strategy: Literal["Random", "RandomNoReplacement", "LabelBalanced", "TriggerBalanced", "Loss", "No"] = Field(
-        description="Strategy used to downsample the data. Available strategies: Loss, Gradnorm, No (no downsampling)."
+    class Config:
```

Comment:
The most idiomatic solution would probably be to create subclasses for every strategy. I'm fine to leave this for now though.

Reply:
yes I agree with you. It's just that we are not about to explore the different downsampling strategy now so I didn't make the effort. Thanks for mentioning it!